### ISSUE TITLE
[minor] Up voting and down voting more than once

### ISSUE THREAD

**Issue by anantr98** [Info442-team-2/friendly-mvp#38]
**Describe the bug**
You can downvote twice, and upvote twice on some of them 

**To Reproduce**
1. Go to "baljeat fan club" under groups
2. Click on the up arrow and down arrow on the first discussion post
3. You can see that you can upvote twice and down vote twice. This might change among different discussion posts. 

**Expected behavior**
Being able to up vote once and down vote only once

**Screenshots**

**Desktop (please complete the following information):**
Any browser

**Smartphone (please complete the following information):**

**Additional context**

**Comment by KeithSRoberts** [comment]
Closing this because it is a duplicate of #32
